### PR TITLE
Add AdminSettingsTest, AboutPageTest and update ResetApplicationTest

### PR DESCRIPTION
**Pull request by kkrawczyk123**
@grzesiek2010  I have pushed AboutPageTest but the second case currently does not work as the icons from AboutListAdapter.java don't have tags. Could you please add it?

### CODE REVIEW

File: collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        new MainMenuPage(rule)
+                .clickOnMenu()
+                .clickAbout()
+                .rotateToLandscape(new AboutPage(rule))
+                .assertOnPage()
+                .scrollToOpenSourceLibrariesLicenses();
+    }
+
+    @Test
+    public void when_openAboutPage_should_iconsBeVisible() {
```
Comment (seadowg) [inline collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java:33]: I'm not sure we need this as an automated test. In my opinion, it's unlikely that the icons would dissapear and if they did it wouldn't a huge problem. Do you think we could just delete this test?

**Review by seadowg — CHANGES_REQUESTED** [review]
Just one comment. Let me know what you think!

File: collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java (hunk 1 above)
Comment (kkrawczyk123) [reply collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java:33]: I think that automation of all cases that are performed manually in the regression cycle is valuable and can shorten the time of performing the whole cycle. It is something that we are checking anyway so why not check that automatically instead?

Comment (seadowg) [reply collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java:33]: Right that's a good point. I think we probably need to start thinking about seperating out `regression` from the other Espresso tests so we can have one faster set to run locally as devs but keep the value of having all these regression test cases automated - potentially running then before PR merge or once a day or something.

**Review by seadowg — APPROVED** [review]

**Comment by seadowg** [comment]
@kkrawczyk123 I'm just going to check these test on Test Lab before merging.

File: collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java (hunk 1 above)
Comment (lognaturel) [reply collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java:33]: > I think we probably need to start thinking about seperating out regression from the other Espresso tests so we can have one faster set to run locally

I think this is a really good idea. Is introducing a new Gradle task that runs the non-regression tests a reasonable way to do this?

Comment (seadowg) [reply collect_app/src/androidTest/java/org/odk/collect/android/regression/AboutPageTest.java:33]: @lognaturel I think there is a way to filter on annotations but not sure about packages. I'll have a look into this.

**Comment by seadowg** [comment]
@kkrawczyk123 I'm seeing failures in `when_openAboutPage_should_iconsBeVisible` on Test Lab:

```java
java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
at org.odk.collect.android.support.matchers.DrawableMatcher$1.matchesSafely(DrawableMatcher.java:25)
at org.odk.collect.android.support.matchers.DrawableMatcher$1.matchesSafely(DrawableMatcher.java:17)
at androidx.test.espresso.matcher.BoundedMatcher.matches(BoundedMatcher.java:73)
at androidx.test.espresso.matcher.ViewMatchers.assertThat(ViewMatchers.java:527)
at androidx.test.espresso.assertion.ViewAssertions$MatchesViewAssertion.check(ViewAssertions.java:105)
at androidx.test.espresso.ViewInteraction$SingleExecutionViewAssertion.check(ViewInteraction.java:425)
at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:288)
at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:272)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at android.os.Handler.handleCallback(Handler.java:790)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:164)
at android.app.ActivityThread.main(ActivityThread.java:6494)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

Everything fine locally?

**Comment by grzesiek2010** [comment]
> Everything fine locally?

No, that's why @kkrawczyk123 asked me to implement required changes in the code:

> @grzesiek2010 I have pushed AboutPageTest but the second case currently does not work as the icons from AboutListAdapter.java don't have tags. Could you please add it?

What I've just done in https://github.com/opendatakit/collect/pull/3605/commits/0e09c7da48e8fb965f18c94a2c1e0fa37a340046

**Pull request merged**